js/PulsarSortableComponent.js

```diff
@@ -145,4 +163,39 @@ PulsarSortableComponent.prototype.updateOrder = function() {
     });
 };
 
+// Create or update "Actions" badge when a table row is selected
+PulsarSortableComponent.prototype.actionsBadge = function() {
+    var component = this,
+        checkedBoxesCount = component.$html.find('.has-badges table tr input.checkbox:checked').length,
+        badge = component.$html.find('.has-badges .btn__group.dropdown span.badge'),
+        itemWording = component.$html.find('.has-badges .btn__group.dropdown .item-wording'),
+        deleteOption = '.has-badges .dropdown__menu li:last-child a';
+
+    if (checkedBoxesCount > 0) {
+        // Remove previous badge
+        badge.remove();
+        // Add badge next to "Actions" dropdown text
+        $('<span class="badge">'+ checkedBoxesCount +'</span>').insertBefore('.has-badges .btn__group.dropdown span.caret');
+
+        if ($(deleteOption).text().indexOf("Delete") === 1) {
```

Comment:
Identifying this delete option by it's text seems fragile to me, could we not use an attribute or class here? You just know 12 months down the line someone will change that to "Remove"